### ISSUE TITLE
Opening window in a subprocess from Spyder

### ISSUE THREAD

**Issue by baudoux** [spyder-ide/spyder#9113]
I asked the question there:
https://stackoverflow.com/questions/55511541/subprocess-in-spyder-matplotlib-and-prints-do-not-work

Running this on a regular console works well, but from Spyder the window does not show up. Prints neither btw.

main.py:

```
import subprocess

subprocess.Popen("test.py", shell=True)

```
test.py:

```
import matplotlib.pyplot as plt

plt.plot([1,3,2])
plt.show()

```
I am using Spyder 3.3.3, python 3.7.3 on windows 10

**Comment by ccordoba12** [comment]
Thanks for reporting @baudoux! We'll fix this in our next release (3.3.5).

**Comment by ccordoba12** [comment]
@dalthviz, please work on this one. The idea is to add an option to `Preferences > IPython Console > Advanced Settings`, in a new section called `Windows adjustments`, that allows users to decide if they want to show or not subprocess cmd windows.
